src/DashboardService.Models.Dto/Requests/Group/Filters/GetGroupFilter.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LT.DigitalOffice.Kernel.Requests;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group.Filters;
+public record GetGroupFilter : BaseFindFilter
+{
+  [FromQuery]
+  public Guid? BoardId { get; set; }
+
+  [FromQuery(Name = "nameIncludeSubstring")]
+  public string NameIncludeSubstring { get; set; }
```

Comment:
here is example - https://github.com/LT-Students/DashboardService/pull/10/files#diff-515f3799edbd1b2cb7330a0c606422492a24754cf0e7fb15d715fc6e54d953bc

you need to place here code that allows you to include related entities

Reply:
++